### PR TITLE
Add scripts in groovy on how to use network modifications.

### PR DESCRIPTION
**Pull request by colinepiloquet**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [x] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*
No

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
Tutorial on how to use network modifications in groovy.

### CODE REVIEW

File: groovy-modifications/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+In this tutorial, you will find two scripts explaining how to add a load and a line with the topology to a network.
+
+The network element is added to the network and the switches are also created to connect the element 
+to a given busbar section with a closed disconnector. Open switches are also added to every parallel busbar sections.
+
+Other similar functions are available to add every kind of injections and branches.
+
+On all the scripts, a node/breaker is imported. The single line diagram of voltage levels S1VL2 and S2VL1 of this network are:
+![S1VL2](.github/nb_network_s1vl2.png "S1VL2")
+![S2VL1](.github/nb_network_s2vl1.png "S2VL1")
```
Comment (miovd) [inline groovy-modifications/README.md:12]: Maybe display the networks in the points below, directly in the examples in which they are used (before/after) for more clarity

File: groovy-modifications/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+# CreateLoadBay.groovy
```
Comment (miovd) [inline groovy-modifications/README.md:13]: Can you add a quick word about the injection order?

File: groovy-modifications/README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+In this script, a node/breaker network is first loaded. Then a load is added on the busbar section BBS1 of the voltage level named S1VL2 and the network is exported.
+It is possible to create a builder with any kind of injection adder and thus create any type of injection.
+
+The voltage level S1VL2 will look like that after the modification:
+
+![S2VL1 with new load](.github/nb_network_with_load.png "S2VL1 with new load")
+# CreateLineBay.groovy
```
Comment (miovd) [inline groovy-modifications/README.md:20]: Can you add a quick word about connectable positions?

**Review by miovd — COMMENTED** [review]
Great! A few remarks about the README

File: groovy-modifications/src/main/groovy/com/powsybl/tutorials/groovy_modifications/CreateLineBay.groovy [hunk 4]
```diff
@@ -0,0 +1,37 @@
+package com.powsybl.tutorials.groovy_modifications
```
Comment (geofjamg) [inline groovy-modifications/src/main/groovy/com/powsybl/tutorials/groovy_modifications/CreateLineBay.groovy:1]: A general comment: this is not Groovy, this Java in Groovy file :-)

File: groovy-modifications/src/main/groovy/com/powsybl/tutorials/groovy_modifications/CreateLineBay.groovy [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+import com.powsybl.iidm.modification.topology.CreateBranchFeederBaysBuilder
```
Comment (geofjamg) [inline groovy-modifications/src/main/groovy/com/powsybl/tutorials/groovy_modifications/CreateLineBay.groovy:3]: All these imports could be hidden and done by script loader

File: groovy-modifications/src/main/groovy/com/powsybl/tutorials/groovy_modifications/CreateLineBay.groovy [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+import com.powsybl.iidm.network.extensions.ConnectablePosition
+import com.powsybl.iidm.network.extensions.FourSubstationsNodeBreakerWithExtensionsFactory
+
+// Load network
+network = FourSubstationsNodeBreakerWithExtensionsFactory.create()
+
+// Create line adder
+lineAdder = network.newLine()
```
Comment (geofjamg) [inline groovy-modifications/src/main/groovy/com/powsybl/tutorials/groovy_modifications/CreateLineBay.groovy:11]: Groovy do not need builders, Groovy has closures.

File: groovy-modifications/src/main/groovy/com/powsybl/tutorials/groovy_modifications/CreateLineBay.groovy (hunk 5 above)
Comment (miovd) [reply groovy-modifications/src/main/groovy/com/powsybl/tutorials/groovy_modifications/CreateLineBay.groovy:3]: Do you think we should add a script loader now or wait for the wrapper we intend to add in core (which would change the scripts I assume)?

Comment (miovd) [reply groovy-modifications/src/main/groovy/com/powsybl/tutorials/groovy_modifications/CreateLineBay.groovy:3]: What we wanted was a script example that can be executed from the iTools `run-script`

File: groovy-modifications/README.md [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+| InjectionAdder         | /                   | The adder corresponding to the injection to be added. <br/>Should be created beforehand and contain all the required information.                                                 |
+| BusOrBusbarSectionId   | /                   | The ID of the busbar (node/breaker network) or of the bus (bus/breaker network) on which the injection should be connected.                                                       |
+| InjectionPositionOrder | /                   | Required in Node/Breaker. The order position of the injection to create the corresponding extension. <br/> The injection will be positioned accordingly on the busbar section.    |
+| InjectionFeederName    | Id of the injection | Only in Node/Breaker. An optional name to be put in the ConnectablePosition extension and that will be displayed on the diagrams.                                                 |
+| InjectionDirection     | BOTTOM              | Only in Node/Breaker. An optional direction to be put in the ConnectablePosition extension and that will correspond to the direction of the injection in the Single line diagram. |
+
```
Comment (miovd) [inline groovy-modifications/README.md:36]: Add here a NB for people to say that the current design is not exactly groovy-friendly...

**Review by miovd — APPROVED** [review]

**Pull request merged**